Formula/srcml.rb

```diff
@@ -0,0 +1,223 @@
+class Srcml < Formula
+  desc "convert source code to XML"
+  homepage "http://www.srcml.org/"
+  url "http://131.123.42.38/lmcrs/beta/srcML-src.tar.gz"
+  version "0.9.5"
+  sha256 "55dd2115548e270724af4251187343656d2dfda0e7d372fee15ae27262e3fa8e"
+
+  depends_on "cmake" => :build
+  depends_on "LibArchive" => :build
+  depends_on "antlr@2" => :build
+  depends_on "libantlr3c" => :build
+  depends_on "Boost" => :build
```

Comment:
Is boost really a build-only dependency? (i.e. we use a header-only part of boost)